### ISSUE TITLE
[metric-learn] Install codecov to have reports in PRs

### ISSUE THREAD

**Issue by bellet** [scikit-learn-contrib/scikit-learn-contrib#51]
(not sure this is the right place to post this but I can't think of somewhere else)

Dear sklearn-contrib admins,

I am one of the maintainers of @scikit-learn-contrib/metric-learn and we would like to install the [Codecov app](https://github.com/marketplace/codecov) to be able to see codecov reports directly on the PRs

However it seems that owner rights on the team (we are only maintainers), or perhaps even on the organization, is needed to install it on https://github.com/scikit-learn-contrib/metric-learn

Could you please give us appropriate rights or simply install it for us? Thanks a lot!
